.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: CI workflow
```

Comment:
name it just ci for convention and reflect the change in the readme file

Reply:
Changed to `ci`